README.md

```diff
@@ -1,7 +1,27 @@
 # The Active Admin Store
-[![Travis CI](https://img.shields.io/travis/activeadmin/activeadmin/master.svg)           ](https://travis-ci.org/activeadmin/activeadmin)
-[![Dependency Status](https://gemnasium.com/badges/github.com/activeadmin/demo.activeadmin.info.svg)](https://gemnasium.com/github.com/activeadmin/demo.activeadmin.info)
+[![Travis CI](https://img.shields.io/travis/activeadmin/activeadmin/master.svg)](https://travis-ci.org/activeadmin/activeadmin)
 
 This is a test application to demo Active Admin.
 
 http://demo.activeadmin.info
+
+
+## Local development setup
+
+Dependencies: Ruby 2.5+ and PostgreSQL
+
+* Clone project using Git
+* Run `bundle install`
+* Configure database:
+  - Create the DB user by running command:
+    ```bash
+    sudo -u postgres psql -c "CREATE USER demo_aa WITH CREATEDB NOCREATEROLE PASSWORD 'demo_aa'"
+    ```
+  - Export the environment to set the credentials with
+    `export DATABASE_USERNAME=demo_aa`
+    and
+    `export DATABASE_PASSWORD=demo_aa`
```

Comment:
@workgena I think it would help to have this be multiline like the creating the DB user command. This way its just copy and paste into a Terminal. 😏 I suggest something like:

```
  - Export the environment to set the credentials with:
     ```
     export DATABASE_USERNAME=demo_aa
     export DATABASE_PASSWORD=demo_aa
     ```
```

Reply:
Yeah, or even `export DATABASE_USERNAME=demo_aa DATABASE_PASSWORD=demo_aa` :)

Reply:
@deivid-rodriguez I was wondering about that! Good to know it can be done. 😏 Then I'll update my suggestion. Thanks!

Reply:
agree, done, and rebase

Reply:
Awesome thanks @workgena! 👍🏻